### PR TITLE
TranslucentStatusBar by default.

### PR DESCRIPTION
**Pull request by chunfengyao**
As been [mentioned](https://github.com/gotify/android/pull/289?notification_referrer_id=NT_kwDOAbm1UrM2MzE4NDE2NTMyOjI4OTQ3Nzk0#issuecomment-1539456821) here, I have removed the settings for TranslucentStatusBar, and make It as default.

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -15,12 +15,28 @@
...
+
+    <style name="AppTheme.NoActionBar.TranslucentStatus">
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+        <item name="android:windowTranslucentStatus">true</item>
+        <!--Android 5.x should set statusBarColor with transparent, or It will be grey-->
+        <item name="android:statusBarColor">@android:color/transparent</item>
+    </style>
+
     <style name="AppTheme.SplashScreen" parent="Theme.SplashScreen">
```
Comment (jmattheis) [inline app/src/main/res/values/styles.xml:27]: Why create a new style with the suffix `.TranslucentStatus` when the settings could be integrated into the existing AppTheme styles?

Comment (chunfengyao) [reply app/src/main/res/values/styles.xml:22]: Perhaps, the existing theme will be referred to later? so, I made a copy.

Comment (chunfengyao) [reply app/src/main/res/values/styles.xml:22]: The new style has been merged into the existing AppTheme in the latest commits.

**Review by jmattheis — APPROVED** [review]
:+1:

**Pull request merged**